### ISSUE TITLE
carousel in home page

### ISSUE THREAD

**Issue by emj365** [xinfab/xinfab.github.io#11]
It's better has this in the home page http://bootstrapdocs.com/v3.3.5/docs/javascript/#carousel.

Any idea?

**Comment by lpgn** [comment]
i like it

**Comment by emj365** [comment]
Few good photos require for that.

**Comment by lpgn** [comment]
we have a bunch but the best would be to be able to let people upload it themselfs like https://www.openthings.wiki/project/list so every time someone finishes a project it will be displayed on our website

**Comment by emj365** [comment]
Carousel used to show Ads or Highlights by the official.
I guess fab-manager can do something like https://www.openthings.wiki/project/list.

**Comment by xiaoeva** [comment]
Having people upload their project pictures themselves would be great (especially if we could link https://www.openthings.wiki/project/list. to the Carousel so it happens automatically). 

But to start things off, I think we can upload some photos so it's not empty. I would love to add a Carousel to the website!

**Comment by emj365** [comment]
Glad to know that.

Carousel works as a slider. Each slider page should include: 
- Impress image as banner
- URL link to blog or page 

We can prepare those first.

**Comment by emj365** [comment]
added http://beta.xinfab.com/

**Comment by emj365** [comment]
@lpgn I need suggestion. Please check the demo.

**Comment by lpgn** [comment]
Demo Checked its a huge upgrade! I would make picture bigger like http://fablab.saul.ie/ That would nice!

**Comment by emj365** [comment]
Good idea @evawxiao said that too. I will try to put it on top the page totally. Then we will need photos in big resolution.

**Comment by lpgn** [comment]
yes

**Comment by emj365** [comment]
I guess you will like the design like this: http://www.human-themovie.org/
But I build v2 based on old design. Small carousel is better than nothing. But I'll try.
Maybe we buy a web template for V3 about 30USD. http://themeforest.net

**Comment by lpgn** [comment]
this looks really beautiful. if you find one let me know and I buy it.
